combat-trainer.lic

```diff
@@ -384,6 +388,11 @@ class LootProcess
     return if Time.now - @loot_timer < @loot_delay
 
     game_state.mob_died = true
+    if (Time.now - @last_rites_timer > 600) && @last_rites
+      bput "pray #{DRRoom.dead_npcs.first}, 'You beseech your god for mercy', 'You pray fervently', 'You continue praying for guidance', 'Quietly touching your lips with the tips of your fingers', 'murmur a brief prayer for its safe travels beyond this life'"
```

Comment:
I think you single and double quotes are out of place. I expected to see something like `bput("pray ...", 'message 1', 'message 2')`.